query/src/main/scala/hmda/query/projections/institutions/InstitutionCassandraProjection.scala

```diff
@@ -0,0 +1,41 @@
+package hmda.query.projections.institutions
+
+import akka.NotUsed
+import akka.actor.{ ActorSystem, Scheduler }
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.cassandra.scaladsl.CassandraSink
+import akka.stream.scaladsl.Source
+import hmda.persistence.messages.events.institutions.InstitutionEvents.{ InstitutionCreated, InstitutionModified }
+import hmda.persistence.processing.HmdaQuery._
+import hmda.query.model.institutions.InstitutionQuery
+import hmda.query.repository.institutions.InstitutionCassandraRepository
+import hmda.query.repository.institutions.InstitutionConverter._
+
+import scala.concurrent.ExecutionContext
+
+class InstitutionCassandraProjection extends InstitutionCassandraRepository {
+
+  def startUp(): Unit = {
```

Comment:
This starts up a stream of events from the journal, yes. The difference from previous implementations (i.e. what happens in validation) is that this stream is running continuously, listening to changes in the journal to push those events to the read model. One part that we haven't done yet (because it's not really 100% clear yet what the requirements are) is storing snapshots as an optimization, so not every event has to be read back to the read model.